### PR TITLE
[Caps/Format] add tensor format in caps

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add tensor format in caps.
Fix pad template with flex tensor and remove old mimetype.
Now caps string for flex-tensor has framerate field.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_common.c [hunk 1]
```diff
@@ -112,9 +112,16 @@ static GstCaps *
 _get_tensors_caps (const GstTensorsConfig * config)
 {
   GstCaps *caps;
+  tensor_format format;
 
   caps = gst_caps_from_string (GST_TENSORS_CAP_DEFAULT);
 
+  format = config->info.info[0].format;
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_common.c:119]: Question: why "format" is in TensorInfo? not in TensorsInfo? Does anyone know why it should be in TensorInfo?

Comment (myungjoo) [reply gst/nnstreamer/tensor_common.c:119]: If it's better be in TensorsInfo, not TensorInfo, it should be dealt with another PR later.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_common.c:119]: Considered multi tensors with mixed format (0:static / 1:flex / ...). I will check this after #3389 (removing tensor-config).

**Review by dongju-chae — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_common.c (hunk 1 above)
Comment (myungjoo) [reply gst/nnstreamer/tensor_common.c:119]: If you want to support mixed formats, ```other/tensors,format=${type}``` won't work.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_common.c:119]: related issue #3405

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM 👍👍

File: gst/nnstreamer/tensor_common.c (hunk 1 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_common.c]: Fix this on https://github.com/nnstreamer/nnstreamer/pull/3418

**Pull request merged**